pages/common/flutter.md

```diff
@@ -1,6 +1,6 @@
 # flutter
 
-> Google’s free, open source, and cross-platform mobile app SDK.
+> Google's free, open source, and cross-platform mobile app SDK.
```

Comment:
Most terminals can render unicode now. Why change it to ascii?

Reply:
To be consistent with other texts. All apostrophes are ASCII, only this one was Unicode.

Reply:
Not only it was inconsistent, but subtle differences like these can also introduce hard-to-diagnose issues when e.g. searching the content.

Reply:
Windows is still pretty useless with most unicode characters. 😉

Reply:
Ah, I see! Thanks for the clarification :-)

Regarding searching though, generally a decent search engine will [transliterate to ascii automagically before indexing](https://starbeamrainbowlabs.com/blog/article.php?article=posts/305-Search-Engine-Optimisation-The-Curious-Question.html).